ci-operator/step-registry/ipi/conf/kubevirt/ipi-conf-kubevirt-commands.sh

```diff
@@ -6,27 +6,29 @@ set -o pipefail
 
 CONFIG="${SHARED_DIR}/install-config.yaml"
 
-
-KUBEVIRT_BASE_DOMAIN="tenant1.origin-on-kubevirt.dev.openshift.com"
-KUBEVIRT_API_VIP=192.168.123.15
-KUBEVIRT_INGRESS_VIP=192.168.123.20
-KUBEVIRT_KUBECONFIG=${HOME}/.kube/config
+KUBEVIRT_BASE_DOMAIN="testci.tenant1.dev.openshift.com"
```

Comment:
But we don't know yet if we can use that DNS name

Reply:
We will change it - this is only for create the workflow
nothing is triggerint it yet